tools/sidekick/config.json

```diff
@@ -1,6 +1,7 @@
 {
   "project": "Milo",
   "host": "milo.adobe.com",
+  "trustedHosts": ["milo.adobe.com", "main--milo--adobecom.hlx.page"],
```

Comment:
would need .live as well

Reply:
Done in [dbab334](https://github.com/adobecom/locui/pull/1/commits/dbab3342bf6f651275461cfb7ee79aeccaa3c8ae) 👍